apps/vscode/command_client/command_client.py

```diff
@@ -197,20 +197,33 @@ def run_command(
     return decoded_contents["returnValue"]
 
 
-def get_communication_dir_path():
-    """Returns directory that is used by command-server for communication
+command_server_directory = None
 
-    Returns:
-        Path: The path to the communication dir
-    """
-    suffix = ""
 
+# NB: See https://github.com/talonhub/community/issues/966 for why we do OS-specific temp dirs
+def get_platform_specific_communication_dir_path():
     # NB: We don't suffix on Windows, because the temp dir is user-specific
     # anyways
+    suffix = ""
     if hasattr(os, "getuid"):
         suffix = f"-{os.getuid()}"
+    if app.platform == "linux" or app.platform == "mac":
+        return Path("/tmp") / f"{actions.user.command_server_directory()}{suffix}"
+    elif app.platform == "windows":
+        # subprocess.run(["attrib","+H","myfile.txt"],check=True)
+        return Path(TALON_HOME / f"{actions.user.command_server_directory()}")
```

Comment:
is `TALON_HOME` guaranteed to never change on Windows? I'd almost be tempted to hardcode it here if we're hardcoding it in https://github.com/pokey/command-server/pull/21 cc/ @lunixbochs

Reply:
I switched it to use hard coded to be consistent. But someone will still need to test it on windows